Makefile

```diff
@@ -73,9 +76,15 @@ deps\:lint:
 deps\:test:
```

Comment:
todo: can introduce a new rule called `deps:test-all` here additionally. Then can add it to the workflow file as suggested in a `todo` below:

```
.PHONY: deps\:test-all
deps\:test-all:
	@$(MAKE) deps:test
	@$(MAKE) deps:lens
```

Reply:
I could, but this seems slightly out of scope and would not provide any immediate benefit. It also seems to clash with `make deps`?